main/modemmanager/template.py

```diff
@@ -47,6 +47,10 @@
 sha256 = "6c8f8720737a3788e394c700f36236278c9de09d76069a079e6f1daaf08b2768"
 
 
+def post_install(self):
+    self.install_service(self.files_path / "modemmanager", enable=True)
```

Comment:
```suggestion
    self.install_service(self.files_path / "modemmanager")
```

shouldn't be enabled by default on install

Reply:
still have to remove this

Reply:
oops